avocado/utils/memory.py

```diff
@@ -29,7 +29,7 @@
 from .data_structures import DataSize
 
 
-class MemoryError(Exception):
+class MemError(Exception):
```

Comment:
Well this has been with us for more than half a year and was added by a user. It's likely used outside so we really should provide transition period and entry to `docs/source/release_notes/lts/next.rst`

Reply:
@ldoktor I've looked at avocado-misc-tests, avocado-vt, etc and found no usage of this exception.  The user who contributed is the author of a number of avocado-mist-tests commits, so it seems like this exception is only used internally in this module.

Apart from that, this is from 'utils', so non-backwards compatible changes are OK (we will document them, of course).  The deprecation process will always be done for the core APIs though.

Reply:
OK I agree. Would you please include the LTS note? (we have to start somewhere and we keep postponing it for 2 sprints already...)

Reply:
... force-update is fine by me